src/analysis/bounded_variation.lean

```diff
@@ -520,6 +530,96 @@ begin
   rw [← evariation_on.union f A B, ← inter_union_distrib_left, Icc_union_Icc_eq_Icc hab hbc],
 end
 
+/-TODO : where should the following two lemmas go? -/
+private lemma monotone_on_of_right_inv_on_of_maps_to_of_monotone_on
```

Comment:
These shouldn't be private: they look useful generally! (And we try to avoid private lemmas even for useless ones, as this breaks documentation and follow to source).